pages/advocates.vue

```diff
@@ -34,4 +41,31 @@ useSeoMeta({
   twitterCard: "summary_large_image",
   twitterDescription: description,
 });
+
+const advocatesJoinLink = {
```

Comment:
We might be able to add the `Link` interface from **types/links.ts** here.

Reply:
Updated in https://github.com/Qiskit/qiskit.org/pull/3443/commits/83cbb3b3030bb36070a9ad5860b1b8d55b12cfe7